component/client/core/ClientChat.tsx

```diff
@@ -28,7 +29,17 @@ class ClientChat extends React.Component<IClientChatProps, IClientChatState> {
     this.messenger = <Messenger handleSend={this.handleSend}/>;
   }
 
+  public componentDidMount() {
```

Comment:
If we are not using redux, this is unnecessary. The componentDidMount was used to ensure that the url was updated each time the component mounted.